skills/naming-register-policy/scripts/newcomer-audit-core.ps1

```
@@ -0,0 +1,682 @@
+        information, so it collapses to a single occurrence for that line.
+        Distinct lines each still surface their own occurrence.
+    #>
+    [CmdletBinding()]
+    [OutputType([object[]])]
+    param(
+        [Parameter(Mandatory)]
+        [string]$Content,
+
+        [Parameter(Mandatory)]
+        [AllowEmptyCollection()]
+        [array]$Matches,
```

PSScriptAnalyzer (1.25.0)

[warning] The Variable 'Matches' is an automatic variable that is built into PowerShell, assigning to it might have undesired side effects. If assignment is not by design, please use a different name.

(PSAvoidAssignmentToAutomaticVariable)